lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/clipboard.rb

```diff
@@ -410,7 +410,14 @@ def download_file( dest_folder, source )
   end
 
   def parse_dump(dump, get_images, get_files, download_path)
-    loot_dir = download_path || "."
+
+    loot_dir = download_path || ::File.expand_path("~/.msf4/clipboard_loot/")
+    # prevent writing into existing directory without file download enabled
+    if ::File.directory?(loot_dir) && !get_files
+      print_error("Cannot write to existing directory if file download is not enabled")
+      return false
+    end
```

Comment:
I think we might want to look at aligning this more with our standard loot storage logic here. https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/auxiliary/report.rb#L398

Unfortunately, I don't think you can directly use that report code because Rex namespace'd code shouldn't be using Msf code. The parts we'd want to take from the Msf code though would be at least the path logic.